webservices/utils.py

```diff
@@ -322,9 +322,9 @@ def get_election_duration(column):
 def get_elasticsearch_connection():
     es_conn = env.get_service(name='fec-api-search')
     if es_conn:
-        es = Elasticsearch([es_conn.get_url(url='uri')])
+        es = Elasticsearch([es_conn.get_url(url='uri')], timeout=30, max_retries=10, retry_on_timeout=True)
```

Comment:
It would be better to do:
```
if es_conn:
    url = es_conn.get_url(url='uri')
else:
    url = 'http://localhost:9200'
es = Elasticsearch([url] timeout=30, max_retries=10, retry_on_timeout=True)
```
so that you don't have to specify the additional parameters in 2 places.